malariagen_data/anoph/dipclust.py

```diff
@@ -561,7 +563,7 @@ def plot_diplotype_clustering_advanced(
         region: base_params.regions,
         heterozygosity: bool = True,
         heterozygosity_colorscale: plotly_params.color_continuous_scale = "Greys",
-        snp_transcript: Optional[base_params.transcript] = None,
+        snp_transcripts: Sequence[base_params.transcript] = [],
```

Comment:
Hi @jonbrenas, this will be a breaking API change. Instead we could leave the parameter name the same but widen the type so it can accept either an individual transcript or a sequence of transcripts.

Reply:
Yes, we had this debate a while back (I don't remember which function it was for) about whether it was OK to break the API to make sure that the parameter names still make sense and (I think) we agreed that it was the case but that it would only be done for a new major version. 

Given that we had a major version released 2 weeks ago, it might make more sense to keep the parameter name unchanged but change its content and do our best to remember to update it for the next major release. Is that the decision?